engine/runtime/src/main/java/org/enso/interpreter/runtime/data/Vector.java

```diff
@@ -38,23 +42,31 @@ private Vector(Object storage) {
     this.storage = storage;
   }
 
+  public static InvokeFunctionNode createValueForIndexNode() {
```

Comment:
We could have `InvokeFunctionNode.buildWithArity(int)` factory method and use it instead....

Reply:
done in [2932377](https://github.com/enso-org/enso/pull/7078/commits/29323772dfb73e623ef6c5d019f509e779a72195)